src/entities/SecurityToken/Issuance/Issuance.ts

```diff
@@ -19,17 +25,7 @@ export class Issuance extends SubModule {
   /**
    * Issue a certain amount of tokens to an address. The address must already have been added via `modifyData`. Otherwise,
    * the corresponding shareholder data for that address must be supplied to this method
-   * NOTE: If shareholder data is supplied, client-side validations to verify if the transfer is possible won't be performed
-   *
-   * @param issuanceData array of issuance data
```

Comment:
This one should still be there (as `args.issuanceData`)

`@param args.issuanceData - array that specifies who to issue tokens to and which amounts. Can also contain KYC data`